Core/Core/Logging/SpeckleException.cs

```diff
@@ -1,4 +1,9 @@
 ﻿using System;
```

Comment:
What i was thunking here was that we don't really need two exception classes, and could just have one! Perhaps it's not worth it though. Here's my 0.002c re what I had in mind: 

```cs

public SpeckleException( string message, bool log = false) { ... } 

public SpeckleException( GraphQLError[ ] gqlErrors, bool log = false) { ... } 

```

Not sure the `GraphQLException` does anything special in the way we handle it, but IIRC, it doesn't?

Reply:
Ofc, internally: 
```cs
public SpeckleException( string message, bool log = false) 
{ 
   // ...
   if(log) 
   { 
     Log.Capture() etc.   
   }
} 
```

I'm not sure if we catch globally unhandled exceptions in .NET?

Reply:
> I'm not sure if we catch globally unhandled exceptions in .NET?

They are usually swallowed by the host applications.